app/src/main/java/com/mapbox/maps/testapp/utils/NavigationSimulator.kt

```diff
@@ -69,17 +81,78 @@ class NavigationSimulator(
   }
 
   private fun initMapboxMap() {
+    routeLayer = lineLayer(ROUTE_LINE_LAYER_ID, GEOJSON_SOURCE_ID) {
+//      lineColor(mapView.context.resources.getColor(R.color.mapbox_blue))
+      lineWidth(
```

Comment:
Looks pretty complicated, maybe drop a comment describing the expression?

Reply:
This is a direct port from navigation sdk, this is a fine-tuned expression from user experience I assume.

Reply:
yes, it visually more appealing when zooming